babel/messages.pot

```diff
@@ -2477,13 +2477,13 @@ msgid "Choose a target"
 msgstr ""
 
 #: superset/assets/javascripts/explore/stores/visTypes.js:828
-msgid "ISO 3166-2 codes of region/province/department"
+msgid "ISO 3166-1 codes of region/province/department"
```

Comment:
I don't think you want to change this

Reply:
sorry since i delected this branch ,so i re commit a PR in #3717